osmnx/downloader.py

```diff
@@ -451,6 +454,8 @@ def _osm_network_download(polygon, network_type, custom_filter):
     else:
         osm_filter = _get_osm_filter(network_type)
 
+    session = _create_session()
```

Comment:
I notice that `_create_session` is called 3 times (i.e. separately in the different "download" functions). Is there a reason to do it this way, rather than creating a session one time, for example whenever `ox.config` is called, and then just using that session throughout?